qubesctl

```diff
@@ -47,6 +53,15 @@ def main(args=None):  # pylint: disable=missing-docstring
                         nargs=argparse.REMAINDER)
     args = parser.parse_args(args)
 
+    if args.sync_extmods:
+        opts = qubessalt.load_opts()
+        # Doing it this way because the RunnerClient requires actual
+        # master opts, which we could load, but we want this to end
+        # up in the minion extension_modules dir. Don't assume defaults,
+        # don't load twice.
+        with ctx.func_globals_inject(saltutil.sync_all, __opts__=opts):
```

Comment:
If this is too much magic, we could iterate over a list of module types to sync and call `salt.utils.extmods.sync` directly (thus emulate what the runner does).

Reply:
This part is okay